HomeWorkSolution/Task1/DAL/MotocycleRepositoryCollection.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Loger;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1.DAL
+{
+    class MotocycleRepositoryCollection : IMotocycleRepository
+    {
+        IList<Motocycle> _motos;
+        private LogWorker loger = new LogWorker();
```

Comment:
for the private stuff -> just add `_` as a fist character like you did with `_motos`